README.md

```diff
@@ -107,6 +107,14 @@ The tests are run via the vendored
 [shunit2](https://github.com/kward/shunit2)
 test framework.
 
+### Debugging
+
+To show the logged build outputs:
```

Comment:
Thanks for adding this! Do you mind adding a little more description about this? Maybe where exactly it should go, or an example snippet within the test?